backend/core/src/applications/services/application-csv-exporter.service.ts

```diff
@@ -132,8 +132,10 @@ export class ApplicationCsvExporterService {
       if (obj[app.application_id] === undefined) {
         if (includeDemographics) {
           demographics = {
-            Ethnicity: app.demographics_ethnicity,
             Race: app.demographics_race.map((race) => this.raceToReadable(race)),
```

Comment:
Seems like this Race to Readable function should be updated since many of the selections are coming back in their key format
<img width="649" alt="Screenshot 2024-02-13 at 5 06 37 PM" src="https://github.com/metrotranscom/doorway/assets/53269332/8776f1b1-3a41-4717-bee8-b5e27617f9fc">

Reply:
on a similar note. does the gender and sexual orientation also need to be parsed on the csv? 
<img width="205" alt="image" src="https://github.com/metrotranscom/doorway/assets/42942267/02639b37-0ddf-4472-b1a2-485f04b0acb1">

Reply:
I'm adding mappings for all of those, thanks.